Data regarding the distribution of OrM in Europe were entered into a database. The data included were composed of OrM fungi isolated from orchid roots, and were georeferenced to the nearest known locality; also, each sample had to have a Genbank accession code in order to obtain the sequences and do the analysis.
Only sequences from well-known OrM were considered, that is: Ceratobasidiaceae, Inocybaceae, Russulaceae, Sebacinaceae, Serendipitaceae, Thelephoraceae and Tulasnellaceae [#dearnaley2012].
Orchid species sampled were *Cephalanthera damasonium* [#julou2005], *Cephalanthera longifolia* [#pecoraro2017],  *Dactylorhiza baltica* [#shefferson2008], *Epipactis atrorubens* [#shefferson2008], *Himantoglossum adriaticum* [#pecoraro2013], *Limodorum abortivum* [#girlanda2005], *Neottia cordata* [#tesitelova2015], *Neottia ovata* [#hansjacquemyn2015, tesitelova2015], *Ophrys bertolonii* [#pecoraro2015], *Orchis anthropophora*[#julou2005], *Orchis militaris* [#shefferson2008], *Orchis purpurea* [#lievens2010], *Orchis simia* [#schatz2010, lievens2010], *Orchis tridentata* [#pecoraro2012], *Spiranthes spiralis* [#duffy2019].

For each georeferenced population six variables were extracted by using the ESDAC database [#panagos2012] and the WorldClim database [#hijmans2005]: Nitrogen, Potassium and Phosphorus soil content, soil pH, minimum temperature of the coldest quarter and maximum precipitation of the wettest month. These variables were selected because mycorrhizal fungi may be sensitive to nutrients in the soil: Nitrogen, Phosphorus and Potassium in high quantities (such as in eutrophicated soils because of agricultural fertilizers) have been seen to cause decline in the belowground mycorrhizal fungi species richness and cause dramatic changes in the community composition and structure [#lilleskov2002, baar2002, grant2011]. Mycorrhizal fungi growth and community composition also seem to be influenced by the soil pH [#aarle2002, carrino-kyker2016], temperature and precipitation [#rillig2003]. Those variables may serve as important proxy for other conditions. Biomes and vegetation are correlated with the environmental conditions, because they change local conditions (like soil pH). Also, human impact can often be seen by the amount of chemicals in the soil, especially close to cultivated fields.
Environmental values were extracted using `GDAL`'s `Sample Raster Values` tool (Using QGIS v. 3.16 as a GUI) and appended to the dataset

![**Geographical distribution of sampling locations of members from OrM families used in this study** (Basemap Imagery ©2021 TerraMetrics, Map data ©2021 GeoBasis-DE/BKG (©2009), Google, Inst Geogr. Nacional)](images/map.png)

## Phylogenetic analysis

In order to understand the distribution and ecology of the OrM we need a better insight of their phylogenesis. The hypothesis was for sequences of the same family to be clustered together, with some doubts regarding the Sebacinales as Serendipitaceae and Sebacinaceae are very closely related and have only been recently separated [#weiss2016].
Phylogenetic analysis were performed on the sequences deposited by the papers included in the database.
The primers used were mainly ITS1F, ITS4, ITS3 and ITS4OF, all targeting regions between the 18S rRNA subunit and the 28S rRNA subunit, including the Internal Transcribed Spacers (ITS hereafter) 1 and ITS 2. Those primers were usually universal for Basidiomycota or in some cases more specific for Tulasnellaceae (like ITS4tul) or other taxa.
Sequence `DQ520100` from *Tremiscus helvelloides* was used as outgroup.

* Sequences were aligned using the MUSCLE algorithm [#edgar2004] and manually trimmed to a visually satisfying overlapping;
* Ugene was used as main GUI, v. 37.0 [#okonechnikov2012];
* Maximum Parsimony analysis was performed using TNT, v. 1.1 [#tnt], using the Tree Bisection and Reconnection algorhithm and with ten replics. 1000 trees were kept and a strict consensus tree was calculated. Bootstrap analysis was performed on the tree with 200 replications to test the validity of the tree. Bootstrap values are displayed as node labels in the appendix tree (Appendix 1);
* the Bayesian analysis (MCMC) was performed using MrBayes, v. 3.2.7a [#huelsenbeck2001], using the Hasegawa-Kishino-Yano with a gamma rate heterogeneity among sites (`lset nst=2 rates=gamma;`). One million trees were generated and sampled each thousand, with four chains running. A final consensus tree was then calculated (see appendix).
* Trees were then visually edited with FigTree v. 1.4.4.
* All parameters are available in the supplemental data, along with the files to reproduce the analysis.

## Multivariate analysis

Before proceeding with the multivariate analysis, sequences have been clustered into Operative Taxonomic Units (OTU hereafter), by using cd-hit v. 4.8.1 [#li2001]. This process yielded 210 OTUs, with the extremes of Serendipitaceae having two OTUs only, and Tulasnellaceae 52 OTUs. 
The database was then pivoted in a presence-absence matrix, and for further analysis it was splitted by family, so that each matrix only had all the OTUs for that single family, yielding seven different matrices. This was necessary to test what variability each family has; another matrix was obtained by grouping together all the observations from the same family, to test the variability between the different OrM families. 

![Number of OTUs from each OrM family](images/clust.png) 


A final matrix was obtained by using the single families/OTUs as rows and removing orchid species as a variable. This was done to understand the  impact of the environmental variables only on each OTU, therefore trying to understand how different the realized niche (i.e. the variance in the environmental variables) is between the groups.

Principal Component Analysis (PCA hereafter) is an orthogonal linear transformation of the data that aims to maximize the variance of the scalar projection of all points of a dataset into a number of axis ordinated by explained variance. This yields a set of axis, which can be used for clustering, to reduce the number of dimensions and perform other analysis. PCA was performed on all matrices using the base R functions `princomp()`, which takes into account the covariance matrix and applies an eigen method of spectral decomposition when possible. Only in the case of Russulaceae, when the number of observation was too limited, `prcomp()` was used, which does a single value decomposition on the centered and scaled data matrix.

As of clustering, Non-metric Multi Dimensional Scaling (NMDS) is another widely used method that allows to visualize the level of similarity of individuals in a dataset. In contrast with PCA, it's non-linear and it's based on a distance matrix, computed by different algorithms depending on the data. It works better with non-parametric data, such as the present one. NMDS was performed on all the matrices by using the R package vegan [#dixon2003a], to understand both how do the OTUs from different families cluster together (if they do) and what environmental factors are most relevant; the Euclidean distance method was used.


In both the PCA and the NMDS we have taken into account how each OTUs presence was influenced by environmental factors, such as climate and soil conditions, and how do they cluster together. 
Species Distribution Models (SDM hereafter) is another conceptual framework we can use to disentangle the assembly processes that lead to the community as we can observe from the data we have, and to infer the relative importance of the environmental factors. SDMs are numerical tools that combine observations of species occurrence or abundance with environmental estimates. They are used to gain ecological and evolutionary insights and to predict distributions across landscapes, sometimes requiring extrapolation in space and time [#elith2009]; those models can also inform us of how species‐to‐species associations depend on the environmental context, in a Joint Species Distribution Model which oftentimes outperforms simple SDMs especially with sparse data [#pollock2014, tikhonov2017].


## Hierarchical Modelling of Species Communities

There are several frameworks to infer patterns of species interactions from environmental data and species co-occurrence patterns[#pollock2014, warton2015]
In the present work, JSDM called Hierarchical Model of Species Community (HMSC) [#ovaskainen2017] was performed by using the `Hmsc` package in R, v. 3.0.9 [#tikhonov2020, hmsc-r2021]; this method uses a Bayesian framework to find the best fitting model based on the data, and works very well with presence-absence data as well as with environmental data [#hefley2016].

All JSDM approaches are based on multivariate probit regression for presence-absence datasets and allow partitioning of shared environmental responses among species from residual correlations among species. The Hierarchical Modelling of Species Communities (HMSC) framework is a joint species distribution model that allows a set of covariates to be modelled on multiple response variables [#ovaskainen2017]. This framework is a hierarchical generalized linear mixed model and is structured by both fixed effects and random effects. This approach allowed us to jointly model the occurrences of mycorrhizal OTUs identified in each individual orchid species as a function of environmental covariates [#tikhonov2017]. Hence, we can estimate to what extent co-occurrence or OrM is explained by environmental variables and taxon interactions for which statistically supported co-occurrence remains after accounting for environmental variables and other factors such as habitat and phylogenetic relatedness of taxa.  

As the response variable (the matrix Y of HMSC; see [#ovaskainen2017]), we used the presence-absence of each of the OTUs. As this matrix is sensitive to outliers, to avoid spurious residual correlations due to the presence of rare OTUs, we reduced the original dataset and included 49 OTUs that occur in more than two different populations. As the data are presence-absence, we applied probit regression to the presence-absence model. We included as fixed effects (the matrix X of HMSC; see [#ovaskainen2017]; which  is the number of species-specific regression parameters to be estimated) soil chemical data extracted from the LUCAS database of the European Soil Data Centre (ESDAC) [#ballabio2019]. HMSC involves a hierarchical structure examining how species responses to environmental covariates depend on phylogenetic relationships. As OrM OTUs were collected across different orchid species which may influence taxon composition, we included orchid species as a random effect in the model. 
We examined the explanatory and predictive powers of our data through species-specific AUC (area under the curve) [#pearce2000] and Tjur’s R2 [#tjur2009] values. Both of these are measures of discrimination, asking how well the occurrence probabilities discriminate sampling units as either occupied or empty. The units of AUC and Tjur R2 are different; a model that behaves ‘as well as by chance’ will yield an AUC of 0.5, whereas for Tjur R2 the same baseline is 0. To compute explanatory power, we made model predictions based on all data. We fitted the full HMSC model with the R-package Hmsc [#tikhonov2020] assuming the default prior distributions (see Chapter 8 of Ovaskainen and Abrego 2020). We fitted the model with 250,000 MCMC iterations comprised of 500 MCMC samples thinned by 500 samples, with the remaining samples thinned by 0.3 samples used as burn-in. We examined MCMC convergence by examining the potential scale reduction factors [#gelman1992] of the model parameters. We considered a co-occurrence between a pair of OrM OTUs well supported if the posterior probability of the association being positive or negative was at least 0.85. To evaluate environmental drivers of differences between taxa and residual co-occurrences, we estimated how much of the community variation was explained by population as a random effect, and the influence of the fixed effects outlined above using the variance partitioning approach of Ovaskainen et al. (2017). To understand how OMF OTUs respond to environmental variables as a function of their phylogeny, we applied the function ‘plotBeta’ to generate a heatmap of OTU niches according to each environmental variable separately, with at least a 0.85 posterior support.